### PR TITLE
[PHP] BasicPHP v0.9.9

### PR DESCRIPTION
**Pull request by ray-ang**

### CODE REVIEW

File: php/basicphp/index.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
```
Comment (waghanza) [inline php/basicphp/index.php:1]: This file should be in a subdirectory name `public`

File: php/basicphp/config.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 framework:
   github: ray-ang/basicphp
-  version: 0.9
+  version: 0.9.9
```
Comment (waghanza) [inline php/basicphp/config.yaml:3]: please avoid patch level

**Review by waghanza — CHANGES_REQUESTED** [review]

File: php/basicphp/index.php (hunk 1 above)
Comment (ray-ang) [reply php/basicphp/index.php]: Thanks. I placed the front controller file inside the public folder.

File: php/basicphp/config.yaml (hunk 2 above)
Comment (ray-ang) [reply php/basicphp/config.yaml]: Thanks. I reverted the line back to version level -> 0.9

**Review by waghanza — APPROVED** [review]

**Pull request merged**